components/server/src/workspace/workspace-starter.ts

```diff
@@ -265,6 +265,7 @@ export class WorkspaceStarter {
                     await this.workspaceDb.trace({ span }).storePrebuiltWorkspace(prebuild)
                     await this.messageBus.notifyHeadlessUpdate({span}, workspace.ownerId, workspace.id, <HeadlessWorkspaceEvent>{
                         type: HeadlessWorkspaceEventType.Aborted,
+                        // TODO: `workspaceID: workspace.id` not needed here? (found in ee/src/prebuilds/prebuild-queue-maintainer.ts and ee/src/bridge.ts)
```

Comment:
@csweichel We've never added a `workspaceID` here, but I see it in all other occurrences of `notifyHeadlessUpdate`. Is it really okay to omit it here? (Couldn't find who listens to this.)